src/content/1.7/modules/core_updates/1.7.5.md

```diff
@@ -45,6 +45,33 @@ public function hookdisplayFooterProduct($params)
 }
 ```
 
+## About getAdminLink and _legacy_link
+
+The 1.7.5 version offers a conversion from legacy links to new migrated Symfony pages. Meaning if you use the `Link::getAdminLink`
```

Comment:
no need to break line, let markdown do it alone :)

Reply:
I know it was juste a matter of readability, but I will write it on one line

Reply:
Your IDE should do it alone. (Let's come to Emacs \o/)
https://github.com/PrestaShop/docs/blob/462b6f79b318876200d5986cc54431e5ceb71a11/src/content/1.7/modules/core_updates/1.7.5.md#about-getadminlink-and-_legacy_link
It is weird in GitHub

Reply:
You mean the last part with `[controller-routing]: {{< relref ...`? Yep in github it does
But I tested it with hugo and it works fine It's used to define the link used before in `[Controller and Routing][controller-routing] page`